{
    "title": "A flaw was found in PoDoFo 0.9.7. An use-after-free in PoDoFo::PdfVecObjects::Clear() function can cause a denial of service via a crafted PDF file.",
    "score": 5.5,
    "severity": "MEDIUM",
    "cwe_id": "CWE-416",
    "cvss_vector": "CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H",
    "cpe_match": [
        "cpe:2.3:a:podofo_project:podofo:0.9.7:*:*:*:*:*:*:*",
        "cpe:2.3:o:fedoraproject:fedora:33:*:*:*:*:*:*:*",
        "cpe:2.3:o:redhat:enterprise_linux:7.0:*:*:*:*:*:*:*"
    ],
    "cve_source": [
        "NVD"
    ],
    "ttpMapping": []
}
